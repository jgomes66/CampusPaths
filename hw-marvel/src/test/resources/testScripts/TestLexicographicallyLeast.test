# Simple test to ensure that the lexicographically least
# path is returned when there are multiple shortest paths

LoadGraph g multipleShortestPaths.csv

FindPath g A C