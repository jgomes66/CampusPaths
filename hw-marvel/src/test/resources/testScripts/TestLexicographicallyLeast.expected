# Simple test to ensure that the lexicographically least
# path is returned when there are multiple shortest paths

loaded graph g

path from A to C:
A to B via BookA
B to C via BookB
